src/app/view/components/_common/dot-global-message/dot-global-message.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable, OnInit } from '@angular/core';
+import { DotMessageService } from '../../../../api/services/dot-messages-service';
+import { DotEventsService } from '../../../../api/services/dot-events/dot-events.service';
+import { DotGlobalMessage } from '../../../../shared/models/dot-global-message/dot-global-message.model';
+
+
+
+/**
+ * Service to provide configurations for Global Messages.
+ * @export
+ * @class DotGlobalMessageService
+ */
+@Injectable()
+export class DotGlobalMessageService implements OnInit {
+    private config: DotGlobalMessage;
+
+    constructor(public dotMessageService: DotMessageService, private dotEventsService: DotEventsService) {}
+
+    ngOnInit(): void {
+        this.dotMessageService
+            .getMessages(['dot.common.message.loading', 'dot.common.message.loaded', 'dot.common.message.error'])
+            .subscribe();
+    }
+
+    /**
+     * Display text messages.
+     * @param {string} message
+     */
+    display(message?: string): void {
+        this.config = {
+            value: message ? message : this.dotMessageService.get('dot.common.message.loaded'),
+            life: 3000
```

Comment:
this must be a class attribute

Reply:
I think the opposite, we don't need ```config``` to be a class property, it doesn't represent anything in this class, we can just do:

```typescript
this.dotEventsService.notify('dot-global-message', {
    value: message ? message : this.dotMessageService.get('dot.common.message.loaded'),
    life: 3000
});
```